src/targets/gpu/CMakeLists.txt

```diff
@@ -33,7 +33,11 @@ if(NOT TARGET MIOpen)
     message(SEND_ERROR "Cant find miopen")
 endif()
 
-set(MIGRAPHX_USE_HIPRTC OFF CACHE BOOL "Use hipRTC APIs")
+if(BUILD_DEV)
+    set(MIGRAPHX_USE_HIPRTC OFF CACHE BOOL "Use hipRTC APIs")
+else()
+    set(MIGRAPHX_USE_HIPRTC ON CACHE BOOL "Use hipRTC APIs")
```

Comment:
Can we  use `rocm_find_program_version` on `amdclang++` ? 

If it is less than 16.0.0  we can `set(ENV{MIGRAPHX_ENABLE_HIPRTC_WORKAROUNDS} True)`

Reply:
That only sets it for cmake time and not runtime. Also, there is performance impact with the workarounds so we don't want to enable it by default.